QuasarHelios2015/src/org/team1540/quasarhelios/Elevator.java

```diff
@@ -91,23 +92,30 @@ public static void setup() {
         Cluck.publish("Elevator Lowering", lowering);
 
         FloatInputPoll main = Mixing.quadSelect(raising, lowering, FloatMixing.always(0.0f), FloatMixing.negate(winchSpeed), winchSpeed, FloatMixing.always(0.0f));
-        FloatInputPoll override = new FloatInputPoll() {
-            @Override
-            public float get() {
-                float f = overrideValue.get();
-                if (atTop.get()) {
-                    f = Math.min(0, f);
-                }
-
-                if (atBottom.get()) {
-                    f = Math.max(0, f);
-                }
-
-                return f;
+        FloatInputPoll override = () -> {
+            float f = overrideValue.get();
+            if (atTop.get()) {
+                f = Math.min(0, f);
             }
+
+            if (atBottom.get()) {
+                f = Math.max(0, f);
+            }
+
+            return f;
         };
+        
         FloatMixing.pumpWhen(QuasarHelios.constantControl, Mixing.select((BooleanInputPoll) overrideEnabled, main, override), winch);
 
+        FloatInput elevatorTimeout = ControlInterface.mainTuning.getFloat("elevator-timeout", 1.0f);
+
+        ExpirationTimer timer = new ExpirationTimer();
+
+        timer.schedule(elevatorTimeout, BooleanMixing.getSetEvent(BooleanMixing.combine(raising, lowering), false));
+        
+        raising.send(timer.getRunningControl());
+        lowering.send(timer.getRunningControl());
```

Comment:
As usual, you probably don't want two separate sources going to any output at the same time. Consider using something like `BooleanMixing.orBooleans`.